### PR TITLE
Select: fix initialInputHeight

### LINKED ISSUE #15581: Select component in the form component size Issue
### Element UI version
2.8.2

### OS/Browsers version
Windows7 64/74.0.3729.131

### Vue version
2.6.10

### Reproduction Link
https://codepen.io/asces/pen/YbZRxJ

### Steps to reproduce
点击 禁用状态改变   

### What is Expected?
点击 禁用状态改变   select 控件大小与设置大小一致

### What is actually happening?
点击 禁用状态改变   select 控件大小与设置大小不一致，强制变成高度为40px的高度

### PR DESCRIPTION
**Pull request by yyjjqq94**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.
Closes  #15581

### CODE REVIEW

File: packages/select/src/select.vue [hunk 1]
```diff
@@ -866,7 +866,8 @@
           small: 32,
           mini: 28
         };
-        this.initialInputHeight = reference.$el.getBoundingClientRect().height || sizeMap[this.selectSize];
+        let input = [].filter.call(reference.$el.childNodes, item => item.tagName === 'INPUT')[0];
```
Comment (junyiz) [inline packages/select/src/select.vue]: ```let input = reference.$el.querySelector('input')``` is better

Comment (WaSheep) [reply packages/select/src/select.vue]: `const input = reference.$el.querySelector('input');` more better

Comment (yyjjqq94) [reply packages/select/src/select.vue]: thanks, but i don't know how to modify this pr

Comment (junyiz) [reply packages/select/src/select.vue]: > thanks, but i don't know how to modify this pr

```
git commit --amend
git push origin dev -f
```

Comment (yyjjqq94) [reply packages/select/src/select.vue]: thanks, but origin branch is deleted

**Pull request closed without merging**